client/src/components/campaign/round-new/round-new.html

```diff
@@ -0,0 +1,165 @@
+                        ng-repeat="method in ['yesno', 'rating', 'ranking']"
+                        value="{{ method }}">
+                      {{ "round." + method | translate }}
+                  </md-option>
+                  </md-select>
+                </md-input-container>
+              </div>
+            </div>
+
+            <h4 ng-if="$ctrl.index === 1">Source Category</h4>
+            <p class="muted" ng-if="$ctrl.index === 1">Category on Wikimedia Commons that gathers all contest images. Example of such category may be eg. "Images from Wiki Loves Monuments 2017 in Ghana".</p>
+            <md-autocomplete flex
+              ng-if="$ctrl.index === 1" required
+              md-input-name="autocompleteField"
+              md-input-minlength="2"
+              md-input-maxlength="500"
+              md-autoselect="true"
+              md-selected-item="$ctrl.category"
+              md-search-text="$ctrl.categorySearchText"
+              md-items="item in $ctrl.searchCategory($ctrl.categorySearchText)"
+              md-item-text="item"
+              md-no-cache="true"
+              md-floating-label="Enter category"
+              md-delay="100">
+              <md-item-template>
+                <span md-highlight-text="$ctrl.searchText">{{ item }}</span>
+              </md-item-template>
+            </md-autocomplete>
+
+            <h4>Directions</h4>
+            <md-input-container class="no-margin">
+              <textarea aria-label="Round directions"
+                ng-model="$ctrl.round.directions"
+                rows="2" ></textarea>
+            </md-input-container>
+          </div>
+          <div flex></div>
+          <div class="round__settings" flex="30">
+            <h4>Round File Settings</h4>
+            <p layout="column" layout-align="start stretch">
+              <span layout="row" layout-align="space-between start"
+                  ng-repeat="(name, value) in $ctrl.round.config">
+                <span ng-if="name !== 'min_resolution'">{{ ::name | translate }}</span>
+                <span ng-if="value === false || value === true">
+                  <md-checkbox
+                    ng-model="$ctrl.round.config[name]"
+                    aria-label="{{ ::name }}">
+                </span>
+                <md-input-container ng-if="name === 'min_resolution'">
+                  <label>Minimal resolution</label>
+                  <input type="number" ng-model="$ctrl.round.config.min_resolution">
+                </md-input-container>
+              </span>
+            </p>
+          </div>
+        </div>
+        
+        <div class="round__jurors"
+            layout="row" layout-align="center stretch">
+          <div flex>
+            <h4>Quroum</h4>
+            <p class="muted">The number of jurors that must vote on each image</p>
+            <md-input-container class="no-margin-bottom">
+              <label>Quorum</label>
+              <input type="number"
```

Comment:
Note, this input should never be smaller than the number of jurors, or it (should) throw an error on the backend...

Reply:
Well, it actually throws the error when you go to activate the round. It's better if we catch this earlier IMHO.